### PR TITLE
Update android tools version.

### PR DESCRIPTION
**Pull request by felipegr6**
Update Android tools version for better compatibility in newer projects.

### CODE REVIEW

**Comment by alexandergoncharov** [comment]
Hi @felipegr6 ,
Thanks for contribution!

**Comment by ruslan-bikkinin** [comment]
HI @felipegr6 and thanks for submitting PR. As react-native-code-push is a react-native plugin, we are synchronizing our android version sdk/tools with the one that RN is using. As you can see, latest version of RN is using android tools v23 and targeting sdk v22:
https://github.com/facebook/react-native/blob/master/ReactAndroid/build.gradle#L247-L253
Also these changes produces breaking changes for our current customers. When RN team will decide to change these settings in the future, we could change them too, but not right now.

File: android/app/build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 27
```
Comment (ruslan-bikkinin) [inline android/app/build.gradle]: Please, see [this comment](https://github.com/Microsoft/react-native-code-push/pull/1305#issuecomment-394613176)

File: android/app/build.gradle [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+    buildToolsVersion "27.0.3"
```
Comment (ruslan-bikkinin) [inline android/app/build.gradle]: Please, see [this comment](https://github.com/Microsoft/react-native-code-push/pull/1305#issuecomment-394613176)

File: android/app/build.gradle [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 27
```
Comment (ruslan-bikkinin) [inline android/app/build.gradle]: Please, see [this comment](https://github.com/Microsoft/react-native-code-push/pull/1305#issuecomment-394613176)

**Comment by max-mironov** [comment]
Good day @felipegr6! That sounds like a good idea and we have tried to do this before (and also have several PRs/issues with it) please see these threads mentioned below for details:
https://github.com/Microsoft/react-native-code-push/pull/950#issuecomment-320167492

Please take a look at this issue describing the problem:
https://github.com/Microsoft/react-native-code-push/issues/733

I believe we can close the PR for now, sorry for any inconvenience that might have caused.
If you have more questions or considerations - we are all ears.

**Pull request closed without merging**